LondonBritishLibrary/orient/BLorient570.xml

```diff
@@ -35,16 +35,6 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         
                     </msIdentifier>
-                    <msContents>
-                        <summary/>
-                        
```

Comment:
I would leave the empty `msContents` with an empty `summary`, then the `summary `will be printed on top for all msParts